src/crawlers/cheerio_crawler.js

```diff
@@ -509,61 +435,10 @@ class CheerioCrawler {
                 else resolve(dom);
             });
             const parser = new htmlparser.Parser(domHandler, { decodeEntities: true });
-            response.on('error', reject).pipe(parser);
+            parser.write(response.body);
```

Comment:
I'd like this to keep the streaming interface. It helps a lot with memory management.